src/index.js

```diff
@@ -18,6 +18,19 @@ import ThemeProvider from './models/ThemeProvider'
 const keyframes = _keyframes(generateAlphabeticName)
 const styled = _styled(_styledComponent(_ComponentStyle(generateAlphabeticName)))
 
+// before fix Bug [Export default] #115
 /* Export everything */
-export default styled
-export { css, keyframes, injectGlobal, ThemeProvider }
+// export default styled
+// export { css, keyframes, injectGlobal, ThemeProvider }
+
+// fix Bug [Export default] #115
+const styledComponent = {}
+Object.keys(styled)
+  .map(ele => (styledComponent[ele] = styled[ele]))
+
+styledComponent.css = css
+styledComponent.keyframes = keyframes
+styledComponent.injectGlobal = injectGlobal
+styledComponent.ThemeProvider = ThemeProvider
+// export default
+export default styledComponent
```

Comment:
With that you're breaking named exports.

```js
import {
  css,
  keyframes,
  injectGlobal,
  ThemProvider,
} from 'styled-components'
```

You now have to write

```js
import styled from 'styled-components'
const {
  css,
  keyframes,
  injectGlobal,
  ThemProvider,
} = styled
```

Note that this is not the same. For further information look at http://stackoverflow.com/questions/33524696/es6-destructuring-and-module-imports